test/controllers/comment_controller_test.exs

```diff
@@ -64,6 +64,28 @@ defmodule CodeCorps.CommentControllerTest do
     test "does not create resource and renders 403 when not authorized", %{conn: conn} do
       assert conn |> request_create(@valid_attrs) |> json_response(403)
     end
+
+    @tag :authenticated
+    test "tracks comment created", %{conn: conn, current_user: current_user} do
```

Comment:
I wonder if it's not better to just do this inside the normal `create` test. Fine to break out some of the complexity into a private function if needed.